src/components/tagList.ts

```diff
@@ -0,0 +1,63 @@
+import FluorineClient from '@classes/Client';
+import Embed from '@classes/Embed';
+import { codeBlock } from '@discordjs/builders';
+import { MessageActionRow, MessageButton, ButtonInteraction } from 'discord.js';
+import r from 'rethinkdb';
+
+export const authorOnly = true;
+
+export async function run(
+    client: FluorineClient,
+    interaction: ButtonInteraction,
+    value: string
+) {
+    const page = Number(value);
+    const tags = await r
+        .table('tags')
+        .getAll(interaction.guild.id, { index: 'guild' })
+        .coerceTo('array')
+        .run(client.conn);
+
+    const chunk = tags.reduce((resultArray, item, index) => {
+        const chunkIndex = Math.floor(index / 4);
+
+        if (!resultArray[chunkIndex]) resultArray[chunkIndex] = [];
```

Comment:
```suggestion
        resultArray[chunkIndex] ||= [];
```
can't be `??=` because that checks for `null` and `undefined`, not `NaN`